### ISSUE TITLE
Export aggregated metrics as a way to reduce the number of metrics

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#1981]
*Installation details*
Scylla version (or git commit hash): 1.5

When using hosted monitoring systems, like hostedgraphite, cost is a function of the number of metrics per second. With per-core metrics, large clusters will generate a lot of metrics per sec.

To reduce the volume of metrics Scylla should:
- export aggregate metrics (for example for reactor load)
- control the exported metrics, allowing the user to disable per-core metrics by default

**Comment by slivne** [comment]
I think we have a way to control metrics (we disable the internal
communication by default) - @amnonh  ?

**Comment by amnonh** [comment]
@tzach Can we close this with https://github.com/scylladb/scylladb/commit/2c0932cc412c682ddf7607fd793c1b1a7ea517be and https://github.com/scylladb/scylladb/commit/f5cc6e2055276c6c386527ac25d53937bf93c415

**Comment by tzach** [comment]
Yes, but please add the results of each PR (here, or in the PR):
What is the reduction (in percent) of the number of metrics reported?

**Comment by mykaul** [comment]
> Yes, but please add the results of each PR (here, or in the PR): What is the reduction (in percent) of the number of metrics reported?

@amnonh - please complete the needful and let's close as completed.
